### DISCUSSION TITLE
Best way to replicate Supabase DB in Bigquery?

### DISCUSSION THREAD

**Discussion by pjerryhu** [supabase/supabase#29601] [General]
Hi folks,

I'm looking to use Google Bigquery as Data Warehouse for my supabase projects. 

Do you have recommendations on what technology I should use to move Supabase table to bigquery?
I will still keep DB tables in Supabase with a reasonable retention strategy. 

Ideally the sync can be both bi-directional so I can recover data back to Supabase when needed. 

Thanks!

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/database/extensions/wrappers/bigquery
https://supabase.com/partners/integrations/stacksync

A couple of links.  Not quite Postgres replication though if that is a requirement.
